controllers/new-page-pane.js

```diff
@@ -1,31 +1,192 @@
-const edit = require('../services/edit'),
-  progress = require('../services/progress');
+var edit = require('../services/edit'),
```

Comment:
I didn't notice that when changing it. I'll keep it in mind when writing the generic version